src/gen/annotation.rs

```diff
@@ -0,0 +1,49 @@
+//!
+//! Create `enum` objects
+//!
+//!
+//! Example
+//! -------
+//! ```
+//! use proffer::*;
+//!
+//! let a = Annotation::from("#![be_cool]");
+//!
+//! let src_code = a.generate();
+//! let expected = "#![be_cool]";
+//! assert_eq!(norm_whitespace(expected), norm_whitespace(&src_code))
+//! ```
+
+use crate::SrcCode;
+use serde::{Deserialize, Serialize};
+
+/// Represents a single Rust annotation to a module, function, etc.
+#[derive(Debug, Clone, Serialize, Deserialize)]
+pub enum Annotation {
+    /// Annotation representing an attribute for an item. ie. `#[foo(bar)]`
+    ItemAttr(String),
+    /// Annotation representing a scoped level attribute. ie. `#![warn(...)]`
+    ScopeAttr(String),
+}
+
+impl<S: ToString> From<S> for Annotation {
```

Comment:
How about TryFrom here? https://doc.rust-lang.org/std/convert/trait.TryFrom.html

Reply:
hmm. That's a good idea, but I'm getting this:
```
  --> src/gen/annotation.rs:30:1
   |
30 | impl<S: ToString> TryFrom<S> for Annotation {
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
   |
   = note: conflicting implementation in crate `core`:
           - impl<T, U> std::convert::TryFrom<U> for T
             where U: std::convert::Into<T>;
```
Seem related to this: https://github.com/rust-lang/rust/issues/50133
Any ideas?

Reply:
Ohhh this sucks 😞 Can you add a comment linking to this issue so we can fix this when its fixed.